src/hooks/useRCAuth4Google.js

```diff
@@ -0,0 +1,74 @@
+import { useState, useContext } from 'react';
+import { useToastBarDispatch } from '@rocket.chat/fuselage-toastbar';
+import RCContext from '../context/RCInstance';
+import { useGoogleLogin } from './useGoogleLogin';
+import { useToastStore, useUserStore, useTotpStore } from '../store';
+
+export const useRCAuth4Google = () => {
+  const [userOrEmail, setUserOrEmail] = useState(null);
+
+  const { signIn, signOut } = useGoogleLogin(
+    process.env.REACT_APP_GOOGLE_CLIENT_ID
+  );
+
+  const { RCInstance } = useContext(RCContext);
+
+  const SetisModalOpen = useTotpStore((state) => state.SetisModalOpen);
```

Comment:
Can we change this to `setIsModalOpen` for consistent casing

Reply:
yeah sure